tests/test_dynamic_mock.js

```diff
@@ -84,11 +136,9 @@ test('asynchronous function gets request headers', function(t) {
       },
     },
     function(err, resp, body) {
-      if (err) {
-        throw err
-      }
-      t.equal(resp.statusCode, 201)
-      t.equal(body, 'GHI')
+      t.error(err)
+      t.equal(resp.statusCode, 200)
```

Comment:
Could you use a non-default status code (e.g. 201) so we know that part of this code path still works?